onnx/onnx.in.proto

```diff
@@ -170,9 +176,13 @@ message NodeProto {
 // a parameterized computation graph against a set of named operators that are
 // defined independently from the graph.
 message ModelProto {
+  // 'ONXF' (reversed as 46 58 4E 4F) 0x46584E4F
+  // This field MUST be present.
+  optional fixed32 magic_prefix = 1;
+
   // The version of the IR this model targets. See Version enum above.
   // This field MUST be present.
-  optional int64 ir_version = 1;
+  optional int64 ir_version = 13;
```

Comment:
why field number 13 (gap between 8 and 13)
also, my understanding was ir_version was previously designed to be the first field parsed. so if
we were to move it, would it make more sense as field 2?

Reply:
This was done because there are other PRs in flight that are using those. The particular number isn't important per se. Is there any particular reason folks want to pick one ir_version sooner? I can renumber things to go in text order if there's a preference for that.